### PR TITLE
Added status colors to mouse without borders documentation

### PR DESCRIPTION
**Pull request by ckirby19**
As part of the work on the ticket for [Issue](https://github.com/microsoft/PowerToys/issues/26640)

### CODE REVIEW

**Comment by ckirby19** [comment]
@microsoft-github-policy-service agree

File: hub/powertoys/mouse-without-borders.md [hunk 1]
```diff
@@ -82,6 +82,23 @@ To enable the service mode, run PowerToys in administrator mode and turn on the
 | Add a firewall rule for Mouse Without Borders | Install a firewall rule for Mouse Without Borders. |
 | Show the original Mouse Without Borders UI | Show the original UI from Mouse Without Borders through the original tray icon. Mouse Without Borders needs to be restarted for it to take effect. |
 
+## Status Colors
+The following colors are used to indicate the connection status to the user when trying to connect to another computer:
+
+| Connection Status | Color    | Hex Code    |
+| :-----: | :---: | :---: |
+| NA | Dark Grey   | #00717171  |
```
Comment (alvinashcraft) [inline hub/powertoys/mouse-without-borders.md]:
```suggestion
| NA | Dark Grey | `#00717171` |
```
Formatting the hex codes as code looks better. Could you do this for each row?

Comment (ckirby19) [reply hub/powertoys/mouse-without-borders.md]: Added now!

File: hub/powertoys/mouse-without-borders.md [hunk 2]
```diff
@@ -82,6 +82,23 @@ To enable the service mode, run PowerToys in administrator mode and turn on the
 | Add a firewall rule for Mouse Without Borders | Install a firewall rule for Mouse Without Borders. |
 | Show the original Mouse Without Borders UI | Show the original UI from Mouse Without Borders through the original tray icon. Mouse Without Borders needs to be restarted for it to take effect. |
 
+## Status Colors
+The following colors are used to indicate the connection status to the user when trying to connect to another computer:
+
+| Connection Status | Color    | Hex Code    |
+| :-----: | :---: | :---: |
+| NA | Dark Grey   | `#00717171`  |
```
Comment (htcfreek) [inline hub/powertoys/mouse-without-borders.md:90]: What is NA?

Comment (alvinashcraft) [reply hub/powertoys/mouse-without-borders.md:90]: "Not Available", I think

**Pull request merged**